modules/monitoring-configuring-shiftstack-remotewrite.adoc

```diff
@@ -0,0 +1,148 @@
+= Remote writing to an external Prometheus instance
+
+Use remote write with both {rh-openstack-first} and {product-title} to push their metrics to an external Prometheus instance.
+
+.Prerequisites
+
+- You have access to an external Prometheus instance.
+- You have administrative access to {rh-openstack} and your cluster.
+- You have certificates for secure communication with mTLS.
+- Your Prometheus instance is configured for client TLS certificates and remote writing.
+- Telemetry is enabled in the {rh-openstack} environment.
+- The Cluster Observability Operator is installed on your cluster.
+- The monitoring stack for your cluster is configured.
+
+.Procedure
+
+. Verify that the telemetry service is running by entering the following command:
++
+[source,shell]
+----
+$ oc -n openstack get monitoringstacks metric-storage -o yaml
+----
++
+The `monitoringstacks` CRD indicates whether telemetry is operating normally.
+
+. Create a secret that is named `mtls-bundle` in the `openstack` namespace that contains HTTPS client certificates for authentication to Prometheus by entering the following command:
++
+[source,shell]
+----
+$ oc --namespace openstack \
+    create secret generic mtls-bundle \
+        --from-file=./ca.crt \
+        --from-file=osp-client.crt \
+        --from-file=osp-client.key
+----
+
+. Open the `controlplane` configuration for editing by running the following command:
++
+[source,shell]
+----
+$ oc -n openstack edit openstackcontrolplane/controlplane
+----
+
+. With the configuration open, replace the `.spec.telemetry.template.metricStorage` section so that {rh-openstack} sends metrics to Prometheus. As an example:
++
+[source,yaml]
+----
+      metricStorage:
+        customMonitoringStack:
+          alertmanagerConfig:
+            disabled: false
+          logLevel: info
+          prometheusConfig:
+            scrapeInterval: 30s
+            remoteWrite:
+            - url: https://external-prometheus.example.com/api/v1/write
+              tlsConfig:
+                ca:
+                  secret:
+                    name: mtls-bundle
+                    key: ca.crt
+                cert:
+                  secret:
+                    name: mtls-bundle
+                    key: ocp-client.crt
+                keySecret:
+                  name: mtls-bundle
+                  key: ocp-client.key
+            replicas: 2
+          resourceSelector:
+            matchLabels:
+              service: metricStorage
+          resources:
+            limits:
+              cpu: 500m
+              memory: 512Mi
+            requests:
+              cpu: 100m
+              memory: 256Mi
+          retention: 1d
+        dashboardsEnabled: false
+        dataplaneNetwork: ctlplane
+        enabled: true
+        prometheusTls: {}
+----
+
+. Create a cluster monitoring config map as a YAML file. The map must include a remote write configuration and cluster identifiers. As an example:
```

Comment:
Steps 1, 2, 3, and 4 run on the OpenShift cluster hosting the RHOSO control plane. This configure RHOSO to send their metrics to an external prometheus.

Steps 5, 6, 7, and 8 run on the tenant's OpenShift cluster. This configures the tenant OpenShift cluster to send their metrics to the same Prometheus instance.

I find it not very clear in the doc as it stands.